docs/README.rst

```diff
@@ -4,56 +4,9 @@
 Scrapy documentation quick start guide
 ======================================
 
-This file provides a quick guide on how to compile the Scrapy documentation.
-
-
-Setup the environment
----------------------
-
-To compile the documentation you need Sphinx Python library. To install it
-and all its dependencies run the following command from this dir
-
-::
-
-    pip install -r requirements.txt
-
-
-Compile the documentation
--------------------------
-
 To compile the documentation (to classic HTML output) run the following command
 from this dir::
 
-    make html
```

Comment:
I don't know about this, it's still working for me. If we're removing these docs we might as well update/remove the Makefile itself.

Reply:
I think just covering the Tox approach in the documentation makes sense, assuming there is no common use case where avoiding Tox usage is needed.

Someone interested in not using Tox should be able to figure things out from `tox.ini`.

Reply:
@elacuesta interesting that `make html` still works for you, when I run it I get a warning saying "hoverxref extension is not fully configured. Tooltips may not work as expected." and indeed the tooltips don't work when viewing the generated docs.

Let me know what you guys want to do, I guess it's one of:

1) Only keep tox approach in the docs (I'll then remove the Makefile).
2) Keep both approaches in the docs.

Reply:
It'd be nice to keep an option to use make, and all the make commands we have for docs, if possible.

Reply:
@kmike done.

Reply:
Is there anything more to do on this? As far as I know it's done.